core/jobs/batch_jobs/exploration_migration_jobs.py

```diff
@@ -244,6 +292,20 @@ def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
             | 'Unwrap ok' >> beam.Map(
                 lambda result_item: result_item.unwrap())
         )
+
+        migrated_exploration_summary_results = (
```

Comment:
Should we return these results down below?

Reply:
Ah, you will first need to transform it using `CountObjectsToJobRunResult`

Reply:
Fixed